[Content] Worker transport mission (#6950)
endless-sky/endless-sky: data/human/human missions.txt

			`	"What are you able to pay me?" you ask.`
			label offer
			`	His eyes drop. "I can pay you after I earn the money. I've just gotta get to <planet> to get work, and then I'll send the money. I just want to earn a living, same as anyone."`
			choice
				`	"Okay, I can take you to <planet>."`
					goto accept
				`	"How do I know you'll pay?"`
					goto question
				`	"Sorry, money up front."`
					goto decline
			label question
			`	"You don't trust me? Just keep walking, then, if you're gonna be like that."`
			choice
				`	"Sorry, money up front."`
					goto decline
				`	"Fine, I can take you to <planet>."`
			label accept
			`	"I'll be on board. There's no rush, honestly. I need a break from all this stress." He takes his luggage into your ship as you prepare for the trip.` 
				accept
			label decline
			`	"I knew it. You're just like everyone else." He storms off, not waiting for a response.`
				decline
	on complete
		event "laborer payment" 400
		conversation

> 			`The worker you picked up from <origin> spent every possible moment of the voyage telling his story. He's had a tough life, full of unlucky setbacks, unfair treatment, and outright persecution from others. You think that you'd feel more sorry for him if you weren't so eager to get his ceaseless negativity off your ship.`
> 			`	He carries his bags to the lowered ramp, then turns back before disembarking. "Thanks for the lift. You're one of the good ones, you know?"`

Reviewer: I said I'd come back to this but realized I didn't during my first review.
```suggestion
			`The worker you picked up from <origin> spent every possible moment of the voyage telling his story. He's had a tough life, full of unlucky setbacks, unfair treatment, and outright persecution from others. You stopped paying attention to what he was saying after a while as he went on for what felt like forever, often repeating himself.`
			`	Upon arriving, he carries his bags to the lowered ramp, then turns back before disembarking. "Thanks for the lift. You're one of the good ones, you know?"`
```

event "fumbling mugger followup timer"

mission "Laborer"
	name "Out of Work"
	description "A man down on his luck needs to get to <destination>. He has promised to send you payment as soon as he has it."
	passengers 1
	source 
		attributes "factory" "mining"
	destination attributes "factory" "mining"
	minor
	to offer
		random < 15
	on offer
		conversation

> 			`	Upon entering the spaceport, you see a young man in laborer's clothing is arguing with a woman in a flight suit.`

Reviewer 2: ```suggestion
			`Upon entering the spaceport, you see a young man in laborer's clothing is arguing with a woman in a flight suit.`
```

				`	"Work dried up on the whole planet?"`
			`	"This planet is messed up. Everybody likes to talk behind your back. I did a job for one of the big mining companies here, but my boss had it out for me. Told everybody I did it wrong, even though they never gave me any training at all. Then he went and put me on the permanent record. So yeah, sooner I get off this dump, the better."`
			label offer
			`	His eyes drop. "I can pay you after I earn the money. I've just gotta get to <planet> to get work, and then I'll send the money. I just want to earn a living, same as anyone."`
			choice
				`	"Okay, I can take you to <planet>."`
					goto accept
				`	"How do I know you'll pay?"`
					goto question
				`	"Sorry, money up front."`
					goto decline
			label question
				`	"You don't trust me? Just keep walking, then, if you're gonna be like that."`
			choice
				`	"Sorry, money up front."`
					goto decline
				`	"Okay, I can take you to <planet>."`
			label accept
			`	"I'll be on board. There's no rush, honestly. I need a break from all this stress."`
				accept
			label decline
				`	"I knew it. You're just like everyone else."`
					decline
	on complete
		conversation

> 			`	The worker you picked up from <source> spent every possible moment of the voyage telling his story. He's had a tough life, full of unlucky setbacks, unfair treatment, and outright persecution from others. You think that you'd feel more sorry for him if you weren't so eager to get his ceaseless negativity off your ship.`

Reviewer 2: ```suggestion
			`The worker you picked up from <source> spent every possible moment of the voyage telling his story. He's had a tough life, full of unlucky setbacks, unfair treatment, and outright persecution from others. You think that you'd feel more sorry for him if you weren't so eager to get his ceaseless negativity off your ship.`
```

			`	He carries his bags to the lowered ramp, then turns back before disembarking.`
			`	"Thanks for the lift. You're one of the good ones, you know?"`
			choice
				`	"You're welcome. Better luck on this world."`
					goto luck
				`	"You have my bank address, right?"`
					goto bank
				`	"I need that money now."`
			`	He shoots you an accused look. "You said I could pay you later. As we discussed, I'll send you the payment when I have it."`
				goto end
			label bank
				`	He shoots you a look of thinly masked annoyance. "Yeah. I'll send you the credits when I have them. Bye, now."`
					goto end
			label luck
				`	He bobs his head at you in farewell. "I'll send you the payment when I have it."`
			label end

> 				`	He turns and walks off.`
> event "laborer payment"

Reviewer 2: ```suggestion
				`	He turns and walks off.`

event "laborer payment"
```

mission "Laborer Payment"
	landing
	source
		government "Republic" "Free Worlds" "Syndicate" "Quarg" "Neutral"
	to offer
		has "event: laborer payment"
	on offer
		payment 7000
		conversation

> 			`Against expectations, you received a payment of <payment> from the laborer that you transported a while back. Seems that he's had better luck finding a job than on his home planet.`

Reviewer 2: Could do some expanding.
First, why is it against expectations? What if I never once doubted he'd pay when he could?
Other than that, "the laborer that you transported a while back" is going to be very vague if a player doesn't get this follow-up right away, for instance if they leave for an alien region. I'm not sure myself how to describe him any better, since there's not much to go on in terms of what he looks like, and he doesn't have a name, so maybe including a message from the worker would help, really play it to be the way he talks the rest of the mission, so it's easy to recognize.

Author: I tried adding a new description, does it work?

Reviewer 2: Yup. Thanks.

event "fumbling mugger followup timer"

mission "Laborer"
	name "Out of Work"
	description "A man down on his luck needs to get to <destination>. He has promised to send you payment as soon as he has it."
	passengers 1
	source 
		attributes "factory" "mining"

> 	destination attributes "factory" "mining"

Reviewer 3: You might want to put a distance limit on the destination.
```suggestion
	destination
		attributes "factory" "mining"
```

					decline
			`	He gives you a once-over, and you see a spark of hope in his eyes. "You're a captain? Please, I need to get to <destination>. Work has dried up here, and I need a fresh start."`
			choice
				`	"And what's your offer?"`
					goto offer
				`	"Work dried up on the whole planet?"`
			`	"This planet is messed up. Everybody likes to talk behind your back. I did a job for one of the big mining companies here, but my boss had it out for me. Told everybody I did it wrong, even though they never gave me any training at all. Then he went and put me on the permanent record. So yeah, sooner I get off this dump, the better."`
			label offer
			`	His eyes drop. "I can pay you after I earn the money. I've just gotta get to <planet> to get work, and then I'll send the money. I just want to earn a living, same as anyone."`
			choice
				`	"Okay, I can take you to <planet>."`
					goto accept
				`	"How do I know you'll pay?"`
					goto question
				`	"Sorry, money up front."`
					goto decline
			label question
				`	"You don't trust me? Just keep walking, then, if you're gonna be like that."`
			choice
				`	"Sorry, money up front."`
					goto decline

> 				`	"Okay, I can take you to <planet>."`

Reviewer 3: I don't think using the same dialogue options as before works.

			label accept

> 			`	"I'll be on board. There's no rush, honestly. I need a break from all this stress."`

Author: I added a sentence, is it better now?

				accept
			label decline

> 				`	"I knew it. You're just like everyone else."`

Reviewer 3: ...and? Kind of an abrupt ending.

Author: Added a more conclusive ending.

					decline
	on complete
		conversation

> 			`The worker you picked up from <source> spent every possible moment of the voyage telling his story. He's had a tough life, full of unlucky setbacks, unfair treatment, and outright persecution from others. You think that you'd feel more sorry for him if you weren't so eager to get his ceaseless negativity off your ship.`

Reviewer 3: Avoid telling the player what they're supposed to be thinking.

Author: I'm not really sure how to alter this so the meaning is intact while avoiding telling the player what they're thinking. Do you have any suggestions?

			`	He carries his bags to the lowered ramp, then turns back before disembarking.`
			`	"Thanks for the lift. You're one of the good ones, you know?"`
			choice
				`	"You're welcome. Better luck on this world."`
					goto luck
				`	"You have my bank address, right?"`
					goto bank
				`	"I need that money now."`
			`	He shoots you an accused look. "You said I could pay you later. As we discussed, I'll send you the payment when I have it."`
				goto end
			label bank
				`	He shoots you a look of thinly masked annoyance. "Yeah. I'll send you the credits when I have them. Bye, now."`
					goto end
			label luck
				`	He bobs his head at you in farewell. "I'll send you the payment when I have it."`
			label end
				`	He turns and walks off.`

> event "laborer payment"

Reviewer 3: This event is never set.

Author: I'm not super familiar with how events work, how can I fix this?

Reviewer 3: In "Smuggler's Den, Part 2":
https://github.com/endless-sky/endless-sky/blob/f7a1bf6463e78972a254fdb3ac5b0ad1612f5782/data/human/human%20missions.txt#L211-L212

Author: I tried fixing it, does it work now?

mission "Laborer Payment"
	landing
	source

> 		government "Republic" "Free Worlds" "Syndicate" "Quarg" "Neutral"

Reviewer 3: Not including "Independent" planets? Also make sure this mission doesn't offer on alien Quarg worlds.

	to offer
		has "event: laborer payment"
	on offer

> 		payment 7000

Reviewer 3: Only 7000 credits?

Author: I tried to replace it with a cost multiplier. I'm not super familiar with how cost multipliers work, so I've probably made some sort of error.

		conversation

> 			`You received a payment of <payment> from the down on his luck laborer that you transported a while back, along with a message. "Hello <last>. Not sure if you remember me. I'm the one that had to deal with my boss bad-talking me to everyone on my home planet. I've been doing better, but my new boss keeps trying to rip me off and come in for overtime." He goes into detail about the sordid working conditions in his new job, but you stop reading after the third paragraph.`

Reviewer 3: ```suggestion
			`You receive a payment of <payment> from the down on his luck laborer that you transported a while back, along with a message. "Hello <last>. Not sure if you remember me. I'm the one that had to deal with my boss bad-talking me to everyone on my home planet. I've been doing better, but my new boss keeps trying to rip me off and come in for overtime." He goes into detail about the sordid working conditions in his new job, but you stop reading after the third paragraph.`
```

			`	He carries his bags to the lowered ramp, then turns back before disembarking.`
			`	"Thanks for the lift. You're one of the good ones, you know?"`
			choice
				`	"You're welcome. Better luck on this world."`
					goto luck
				`	"You have my bank address, right?"`
					goto bank
				`	"I need that money now."`
			`	He shoots you an accused look. "You said I could pay you later. As we discussed, I'll send you the payment when I have it."`
				goto end
			label bank
				`	He shoots you a look of thinly masked annoyance. "Yeah. I'll send you the credits when I have them. Bye, now."`
					goto end
			label luck
				`	He bobs his head at you in farewell. "I'll send you the payment when I have it."`
			label end
				`	He turns and walks off.`

event "laborer payment"

mission "Laborer Payment"
	landing
	source
		government "Republic" "Free Worlds" "Syndicate" "Independent" "Neutral"
	to offer
		has "event: laborer payment"
	on offer

> 		payment 10000 3000

Reviewer 3: The payment for this mission is calculated solely based on this mission, which has a distance of 0 and a combined cargo and bunks of 0. Multipliers won't work here, only a constant reward will. (The multiplier you've used is also way too high, but that's beside the point.)

Author: Made it a constant, and upped the payment a bit. Does it look good?

event "fumbling mugger followup timer"

mission "Laborer"

> 	name "Out of Work"
> 	description "A man down on his luck needs to get to <destination>. He has promised to send you payment as soon as he has it."
> 	passengers 1
> 	source 
> 		attributes "mining"
> 	destination
> 		attributes "mining"
> 	minor

Reviewer: ```suggestion
	minor
	name "Out of Work"
	description "A man down on his luck needs to get to <destination>. He has promised to send you payment as soon as he has it."
	passengers 1
	source 
		attributes "mining"
	destination
		attributes "mining"
```

			choice
				`	(Go see what the laborer is arguing about.)`
				`	(Ignore them.)`
					decline
			`	The man is nearly shouting. "Lady, I'm not trying to low-ball you here! I don't have the money! That's the whole point!"`
			`	"I appreciate your conundrum," replies the woman, her voice clipped. "However, my policy is cash upfront. Good day."`
			`	She turns and walks away briskly, and the man slumps down into a chair.`
			choice
				`	"What was that all about?"`
				`	(Leave before he notices you.)`
					decline
			`	He gives you a once-over, and you see a spark of hope in his eyes. "You're a captain? Please, I need to get to <destination>. Work has dried up here, and I need a fresh start."`
			choice
				`	"What are you able to pay me?"`
					goto offer
				`	"Work dried up on the whole planet?"`
			`	"This planet is messed up. Everybody likes to talk behind your back. I did a job for one of the big mining companies here, but my boss had it out for me. Told everybody I did it wrong, even though they never gave me any training at all. Then he went and put me on the permanent record. So yeah, sooner I get off this dump, the better."`

> 			`You say, "What are you able to pay me?"`

Reviewer: ```suggestion
			`	"What are you able to pay me?" you ask.`
```

			label offer
			`	His eyes drop. "I can pay you after I earn the money. I've just gotta get to <planet> to get work, and then I'll send the money. I just want to earn a living, same as anyone."`
			choice
				`	"Okay, I can take you to <planet>."`
					goto accept
				`	"How do I know you'll pay?"`
					goto question
				`	"Sorry, money up front."`
					goto decline
			label question

> 				`	"You don't trust me? Just keep walking, then, if you're gonna be like that."`

Reviewer: ```suggestion
			`	"You don't trust me? Just keep walking, then, if you're gonna be like that."`
```

			choice
				`	"Sorry, money up front."`
					goto decline
				`	"Fine, I can take you to <planet>."`
			label accept
			`	"I'll be on board. There's no rush, honestly. I need a break from all this stress." He takes his luggage into your ship as you prepare for the trip.` 
				accept
			label decline

> 				`	"I knew it. You're just like everyone else." He storms off, not waiting for a response.`
> 					decline

Reviewer: ```suggestion
			`	"I knew it. You're just like everyone else." He storms off, not waiting for a response.`
				decline
```

	on complete
		event "laborer payment" 400
		conversation

> 			`The worker you picked up from <source> spent every possible moment of the voyage telling his story. He's had a tough life, full of unlucky setbacks, unfair treatment, and outright persecution from others. You think that you'd feel more sorry for him if you weren't so eager to get his ceaseless negativity off your ship.`

Reviewer: ```suggestion
			`The worker you picked up from <origin> spent every possible moment of the voyage telling his story. He's had a tough life, full of unlucky setbacks, unfair treatment, and outright persecution from others. You think that you'd feel more sorry for him if you weren't so eager to get his ceaseless negativity off your ship.`
```

> 			`	He carries his bags to the lowered ramp, then turns back before disembarking.`
> 			`	"Thanks for the lift. You're one of the good ones, you know?"`

Reviewer: ```suggestion
			`	He carries his bags to the lowered ramp, then turns back before disembarking. "Thanks for the lift. You're one of the good ones, you know?"`
```

			choice
				`	"You're welcome. Better luck on this world."`
					goto luck
				`	"You have my bank address, right?"`

> 					goto bank
> 				`	"I need that money now."`
> 			`	He shoots you an accused look. "You said I could pay you later. As we discussed, I'll send you the payment when I have it."`
> 				goto end
> 			label bank

Reviewer: I feel like this choice could just be removed. Seems rather pointless because, as he said, you agreed to the payment coming later.
```suggestion
```

> 				`	He shoots you a look of thinly masked annoyance. "Yeah. I'll send you the credits when I have them. Bye, now."`
> 					goto end
> 			label luck
> 				`	He bobs his head at you in farewell. "I'll send you the payment when I have it."`
> 			label end
> 				`	He turns and walks off.`

Reviewer: ```suggestion
			`	He shoots you a look of thinly masked annoyance. "Yeah. I'll send you the credits when I have them. Bye, now."`
				decline
			label luck
			`	He bobs his head at you in farewell. "I promise I'll send you the payment when I have it. Bye, now."`
```

event "laborer payment"

mission "Laborer Payment"
	landing
	source
		government "Republic" "Free Worlds" "Syndicate" "Independent" "Neutral"
	to offer
		has "event: laborer payment"
	on offer
		payment 17000
		conversation

> 			`You receive a payment of <payment> from the down on his luck laborer that you transported a while back, along with a message. "Hello <last>. Not sure if you remember me. I'm the one that had to deal with my boss bad-talking me to everyone on my home planet. I've been doing better, but my new boss keeps trying to rip me off and come in for overtime." He goes into detail about the sordid working conditions in his new job, but you stop reading after the third paragraph.`

Reviewer: ```suggestion
			`You receive a payment of <payment> from the down-on-his-luck laborer that you transported a while back, along with a message. "Hello <last>. Not sure if you remember me. I'm the one that had to deal with my boss bad-talking me to everyone on my home planet. I've been doing better, but my new boss keeps trying to rip me off and make me come in for overtime." He goes into detail about the sordid working conditions in his new job, but you stop reading after the third paragraph.`
```
